### ISSUE TITLE
[0.4.4] no function clause matching in ScoutApm.TrackedRequest.save/1

### ISSUE THREAD

**Issue by seantanly** [scoutapp/scout_apm_elixir#62]
scout_apm version: 0.4.4

```
lib/scout_apm/tracked_request.ex in ScoutApm.TrackedRequest.save/1 at line 157
arg0 | :error

lib/scout_apm/plugs/controller_timer.ex in ScoutApm.Plugs.ControllerTimer.before_send/1 at line 24
```

**Comment by itsderek23** [comment]
Apologies @seantanly - what versions of Phoenix and Elixir are you using?

/cc @mitchellhenke

**Comment by seantanly** [comment]
Phoenix 1.3.0 with Elixir 1.6.5

**Comment by dlanderson** [comment]
Fixed in #63 (0.4.5)
